test/lib/index.test.ts

```diff
@@ -18,16 +18,46 @@ describe('issuePathToLineNumber', () => {
 });
 
 describe('parseFileContent', () => {
-  it('Throws an error if it contains \\/', () => {
+  it('Throws an error if no file type provided it contains \\/', () => {
```

Comment:
Did you mean to write  'Throws an error if no file type provided **and** it contains \\/' here ?